src/pages/therapist.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { withTranslation } from "next-i18next";
+import Button from "@/components/ui/Button";
+import Input from "@/components/ui/Input";
+import Image from "next/image";
+import PageIntro from "@/components/PageIntro";
+function therapist({ t }) {
```

Comment:
Name better to start in capital `function Therapist(){}`

Reply:
I think i have problem with my vs code it change the letter , if you review the test file you will see it capital .